### DISCUSSION TITLE
sshfs working for you?

### DISCUSSION THREAD

**Discussion by DocMAX** [termux/termux-app#3789] [General]
is sshfs working for your? non-root?
```
u0_a191@jelly2 ~ $ sshfs pve:/mnt/pve/data archlinux/mnt/pve/data -o follow_symlinks -o idmap=user -o uid=$(id -u) -o gid=$(id -g)
fuse: failed to open /dev/fuse: Permission denied
```
chmodding /dev/fuse doesn't make it work either... Bad system call

**Comment by agnostic-apollo** [comment]
Mounting requires root.

**Comment by DocMAX** [comment]
cant get it to work even with root. mount command works, but i can't access the folder...

**Reply by agnostic-apollo** [reply]
You need to mount with global namespace and create additional `bindfs` mounts if want other apps to access it.

https://www.reddit.com/r/termux/s/N2IxHKjcZF

https://www.reddit.com/r/termux/s/kjkXt8nVi1

**Reply by 4lx4** [reply]
Wrong info, `allow_other` does not allow any app on android without storage permissions to access mnt point, allow_other works on Unix-level layer, while android apps and permissions work on their higher layer, so for them it's just another usual folder on sdcard.

**Comment by 4lx4** [comment]
@DocMAX , yes, it works nicely. See here: https://github.com/bobrofon/easysshfs/issues/29#issuecomment-4703438557
